test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/config"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type Manager struct {
+}
+
+func (m *Manager) GetConfig(ctx context.Context, c *kubernetes.Clientset, pod *corev1.Pod, containerName string) (*config.Config, error) {
```

Comment:
Similary we can get another Get method which will return interface files from the Node , specifically we need to see if these interface files have right values in pods scope

Reply:
@mrniranjan What do you mean by interface files?

Reply:
ah sorry i commented here,  any how by interface files i mean cpuset.cpus (is an interface file ) for the cpuset controller , just a technical name. 
cgroup-> Controllers(cpuset, cpu, memory, hugetlb, pids) and files inside these controller files are called interface files